### ISSUE TITLE
Create Storyboard

### ISSUE THREAD

**Issue by vanbelle** [Pengwin-man/CMPUT301W15T12#1]
- [ ] login page
- [ ] add new claim
- [ ] edit existing claim
- [ ]  add new expense item
- [ ] edit expense item
- [ ] delete claim

**Comment by IchigoKIl** [comment]
Updated wiki with delete claim and expense item use cases

**Comment by qsjiang** [comment]
# Some Important Points
  - Don't confuse "use cases" with "story board". 
  > "Use case" is a part of the "1. Requirements Specification" which contains detailed textual use cases and test codes

  > "Storyboard" is a part of the "2. User Interface Mockup and Storyboard". It requires diagrams of the user interface and the sequence showing user action between those interfaces.
  
  > details at: https://eclass.srv.ualberta.ca/mod/assign/view.php?id=1356645
  
  - After reading the Project Part 2 requirements, It looks like we need to put "story board" inside the wiki, and attach the image. **BUT for now lets just put everything inside google drive as we discussed on Thursday we will discuss this more on the next meeting.**
  - Github Wiki takes a commonly used doc format called **Markdown**.  So make sure you use markdown when writing the wiki otherwise it's just going to loose all its formating. 
  > Here is an amazing tool for writing markdown. http://dillinger.io/

**Comment by qsjiang** [comment]
- Finished the storyboard for "take a photo".  
> https://docs.google.com/a/ualberta.ca/presentation/d/1yi42dwFf6hozSL-nLe2JjKNcwd6NaqfqD86828pmJ90/edit#slide=id.g626b4e2ab_00

**Comment by qsjiang** [comment]
- Finished the storyboard for "take a photo". 

 > https://docs.google.com/a/ualberta.ca/presentation/d/1yi42dwFf6hozSL-nLe2JjKNcwd6NaqfqD86828pmJ90/edit#slide=id.g626b4e2ab_00

**Comment by IchigoKIl** [comment]
Updated storyboard with delete claim/expense item 

![Delete Expense Item](https://cloud.githubusercontent.com/assets/8731038/6095169/aec9a3b6-af0b-11e4-971c-c9d78b6d7262.png)

![Delete Claim](https://cloud.githubusercontent.com/assets/8731038/6095167/98c5e1f6-af0b-11e4-9383-5744d2bdf806.png)

**Comment by LeahGabrielle** [comment]
Add/Edit an item is up in slides. There's a few slides so I won't put them here.

**Comment by vanbelle** [comment]
For the login page, how do we want to set that up? do they sign in with an email address or a username? Do you want it to be password protected, if so , should we have a function for forgotten passwords? what do you guys think?

**Comment by megsum** [comment]
I think we should have it as an email address because either way they will have to put in an email address to send them things. Also, the forgot passwords is a good idea, I think it goes above what the project wants and he might like it!.

PS: Just working on my claim add/ edit activities right now, will be done by tonight!

**Comment by Pengwin-man** [comment]
I agree with the email address.  However I think it is to early to think of additional features.  Let us do the basics first and if we still have time we will worry about the additional or bonus features
